app/views/shared/_unfinished_task_list.html.slim

```diff
@@ -1,3 +1,4 @@
+- return if tasks.blank?
```

Comment:
we should rather do check in view when we call this partial whether we should render it or not

Reply:
yeah.. makes sense.

Reply:
why do we even need to check whether tasks exists? all of them are being displayed in loop so if array is empty no task will be visible anyway... I removed conditions for now

Reply:
it was actually failing due to the nil error on my system, so I added it.